### ISSUE TITLE
How to get MAC to IP mapping using pyvmomi

### ISSUE THREAD

**Issue by munib07** [vmware/pyvmomi-community-samples#255]
Hi, Is there any way we can have MAC to IP mapping. I mean if a VM has 2 MAC (2 NICs), and each one of them has been allocated different IP. Is it possible to know which IP is associated with which MAC, using pyvmomi ?

**Comment by prziborowski** [comment]
Yes, as long as vmware-tools is installed on the VM and exposes the necessary information to vSphere.
I will assume "vm" is your VirtualMachine managed object.

It is under GuestInfo -> NicInfo\[\] (contains macAddress) -> IpConfigInfo -> IpAddress\[\] (contains ipAddress, both ipv4 and ipv6)

A bit of a more verbose snippet:
```
for nic in vm.guest.net:
   print "Mac: %s" % nic.macAddress
   for ipAddress in nic.ipConfig.ipAddress:
      print "IP Address: %s" % ipAddress.ipAddress

```
